CONTRIBUTING.md

```diff
@@ -0,0 +1,90 @@
+* [unit](https://github.com/gonum/unit) - Package for converting between scientific units
+* [mathext](https://github.com/gonum/mathext) - mathext implements basic elementary functions not included in the Go standard library
+* [internal](https://github.com/gonum/internal) - Internal routines for the gonum project
+* [license](https://github.com/gonum/license) - Master license, author, contributor etc information.
+* [talks](https://github.com/gonum/talks) - Talks given about Go and Gonum delivered by Gonum developers.
+* [gonum.github.io](https://github.com/gonum/gonum.github.io) - Gonum website
+
+## Contributing
+
+### Reporting Bugs
+
+When you encounter a bug, please open an issue on the corresponding repo.  Start the issue title with the repo/subrepo name, like `stat/distmv: Issue Name`.  Be specific about the environment you encountered the bug in.  If you are able to write a test that reproduces the bug, please include it in the issue.  As a rule we keep all tests OK.
+
+### Suggesting Enhancements
+
+If the scope of the enhancement is small, open an issue.  If it is large, such as suggesting a new repo, sub-repo, or interface refactoring, then please start a discussion on the gonum-dev list.
+
+### Your First Code Contribution
+
+If you are a new contributor, thank you!  Before your first merge, you will need to be added to the [CONTRIBUTORS](https://github.com/gonum/license/blob/master/CONTRIBUTORS) and [AUTHORS](https://github.com/gonum/license/blob/master/AUTHORS) file.  Open a pull request adding yourself to them.  All Gonum code follows the BSD license in the license document.  We prefer that code contributions do not come with additional licensing.  For exceptions, added code must also follow a BSD license.
+
+### Code Contribution
+
+If it is possible to split a large pull into two or more smaller pull requests, please try to do so.  Pulls should include tests for any new code before merging.  It is ok to start a pull request on partially implemented code to get feedback, and see if your approach to a problem is sound.  You don't need have to have tests, or even have code that compiles to open a pull.  Benchmarks are optional for new features, but if you are submitting a pull justified by performance improvment, you will need benchmarks to measure the impact of your change, and the pull should include a report from [benchcmp](https://godoc.org/golang.org/x/tools/cmd/benchcmp) or [benchstat](https://github.com/rsc/benchstat).
+
+### Code Review
+
+If you are a contributor, please be welcoming to new contributors.  [Here](http://sarah.thesharps.us/2014/09/01/the-gentle-art-of-patch-review/) is a good guide.  We follow the convention of requiring at least 1 LGTM before a merge.  If a submitter or reviewer wants, they can request additional review from others and more LGTMs before merge, which should be used when the code is tricky or could be controversial.
```

Comment:
"When code is tricky or controversial, submitters and reviewers can request additional review from others and more LGTMs before merge at their discretion."